packages/autometrics/src/utils.ts

```diff
@@ -95,7 +95,8 @@ function getWrappedFunctionPath(): string | undefined {
         !file.includes("autometrics/index.cjs") &&
         !file.includes("autometrics/index.mjs") &&
         !file.includes("wrappers.ts") &&
-        !file.includes("wrappers.js")
+        !file.includes("wrappers.js") &&
+        !file.includes("reflect-metadata")
```

Comment:
i think this would be worthwhile to (briefly) explain with a comment, since the rest of the conditions here pertain to ignoring autometrics library internals, and this condition is specific to the reflect-metadata module